app/renderer/components/preferences/paymentsTab.js

```diff
@@ -317,7 +317,11 @@ const styles = StyleSheet.create({
   },
 
   switchWrap: {
-    width: paymentStyles.width.tableCell
+    width: paymentStyles.width.tableCell,
+
+    // TODO: Remove this when #10913 is merged to master
+    position: 'relative',
+    left: '1rem'
```

Comment:
why is relative unit preferred?

Reply:
it's a magic number. It should work for now on 0.19 and 0.20.

Reply:
ok thanks